clients/web/src/routes/auth/login.svelte

```diff
@@ -26,7 +26,9 @@
         $session.user = extractJwt<SessionUser>(tokens[0]);
         $session.token = tokens[0];
 
-        cookies.set(constants.auth_cookie_key, tokens[0], {expires: 0.25}); // 6 hours
+	// It is important that the auth cookie lasts as long as the refresh token. Do not shorten it, 
```

Comment:
I don't really agree with that but alright. If the jwt expires, we should be checking that a refresh token exists when a regular token doesn't, not having a stale/expired auth token.

Reply:
I forgot to mention this in my PR comments, but I'm ok with refactoring that behavior to incorporate what you had done with the draft PR. I just set things up this way for minimal changes while debugging/testing.

Reply:
I don't particularly care either way, just don't like the feeling of keeping the stale token around. If it works, it works though.

Reply:
Yeah I don't see why we would keep the expired token around for this long 🤔

Reply:
It's only for the purpose of the check (is token expired) in hooks. Can't check a token's expiry if the cookie it lives in is deleted (due to being expired itself). Hyper has a draft PR up that inverts that check and makes this OBE, but we just haven't integrated it yet. I can do that if we'd prefer it.

Reply:
Changed, see latest commit